Sources/AppStoreConnectCLI/Commands/TestFlight/PreReleaseVersion/ReadPreReleaseVersionCommand.swift

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import ArgumentParser
+
+struct ReadPreReleaseVersionCommand: CommonParsableCommand {
+    static var configuration = CommandConfiguration(
+        commandName: "read",
+        abstract: "Get information about a specific prerelease version.")
+
+    @OptionGroup()
+    var common: CommonOptions
+
+    @Argument(
+        help: ArgumentHelp(
+            "The app AppStore ID. eg. 432156789 or app bundle identifier. eg. com.example.App",
+            discussion: "Please input either app id or bundle Id",
+            valueName: "app-id / bundle-id"
+        ),
+        transform: Identifier.init
+    )
+    var identifier: Identifier
+
+    @Argument(
+        help: ArgumentHelp(
+            "The version number of the prerelease version of your app.",
+            discussion: "Please input a specific version no"
+        )
+    )
+    var version: String
+
+    enum Identifier {
```

Comment:
I like this.  Is it something we could apply elsewhere too? Are there other commands that would benefit?

Reply:
Do you mean the enum identifier? There will be a separate PR to refactor this. These are the list of commands that use an identifier ( app id/ bundle id)
ListBetaGroupsCommand
ReadAppCommand
ListBetaTestersCommand
ListPreReleaseVersionsCommand
ReadPreReleaseVersionCommand